src/inst/opts/consul.asc

```diff
@@ -20,6 +20,8 @@ profiles::consul::server::subservices:
       timeout: '1s'
 ----
 
+This defined two services, `http` and `web-system`, with there associated check parameters.
```

Comment:
I find it weird to get the purpose of the yaml code _after_ it. I think it is more clear if this sentence is _before_ the yaml code.

Reply:
Pls check latest update...